web/templates/web/base.html

```diff
@@ -18,11 +18,13 @@
     {% block head %}
     {% endblock %}
     <script src="https://use.fontawesome.com/41c082d3c2.js"></script>
+    <link rel="stylesheet" href="http://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css" />
```

Comment:
Might be worth pulling these from https://cdnjs.com. I think that's OSF's default place to borrow things from.